sentry-logback/src/test/kotlin/io/sentry/logback/SentryAppenderTest.kt

```diff
@@ -305,4 +306,11 @@ class SentryAppenderTest {
             }, anyOrNull())
         }
     }
+
+    @Test
+    fun `does not initialize Sentry when DSN is null`() {
```

Comment:
on Android or standard `Sentry.init`, if no DSN is passed (null), we throw an exception, because "" empty DSN means disabled SDK, are we allowing to do differently on logback?

Reply:
I fixed the test name, as we are not really handling null here. Logback has this weird behavior, that when the env is not set it gets a value "ENV_NAME_IS_UNDEFINED" and the only way to handle it properly is to write it like `${PROPERTY:- }`.

We can either clarify it in the docs or make it easier for users. This has been roughly discussed with @bruno-garcia - would be nice if he also has a quick look